microsoft-edge/toc.yml

```diff
@@ -632,6 +632,10 @@
             - name: Customize keyboard shortcuts  # added article
               href: devtools-guide-chromium/customize/shortcuts.md
               displayName: Settings, Shortcuts
+
+            - name: Extend DevTools with extensions
```

Comment:
@Reezaali I couldn't remember what you mentioned to me the other day about this title? You had a good idea for how to rephrase it.